assembl/static2/css/views/idea.scss

```diff
@@ -12,8 +12,11 @@
   }
 
   .level {
-    width: 600px;
-    margin: 0 auto;
+    width: 925px;
```

Comment:
here again with this 925px width... what is it ?

Reply:
I used the old way and just replace 600px by 925px, the new width of a post everywhere it was mentioned